src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -289,6 +289,27 @@ protected function sortServiceIds(array $serviceIds)
         return $serviceIds;
     }
 
+    public function sortTaggedServicesByPriority(array $services): array
+    {
+        $maxPriority = [];
+        foreach ($services as $service => $tags) {
+            if (!isset($maxPriority[$service])) {
```

Comment:
this won't ever be set here

Reply:
what do you mean ?

Reply:
`isset($maxPriority[$service]` will always be false here, as each iteration of the loop only sets the maximum priority for the current service, which is the key in the input array (and so you never process twice the same service).

Reply:
thanks, I updated this part